app/views/foods/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Food Search Results</h1>
+<% @food_response.first(10).each do |food|%>
+<section class='food_item'>
+  <h2 class='gtinUpc'>Food GTIN/UPC: <%=food[:gtinUpc]%></h2>
+  <h2 class='description'>Food Description: <%=food[:description]%></h2>
+  <h2 class='brandOwner'>Food Brand Owner: <%=food[:brandOwner]%></h2>
+  <h2 class='ingredients'>Food Ingredients: <%=food[:ingredients]%></h2>
+</section>
```

Comment:
So here, `@food_response` is the nearly the entire JSON response. When you refactor your code to use a PORO, then you'd be able to iterate through food ruby objects.